ddsrecorder/src/cpp/user_interface/arguments_configuration.cpp

```diff
@@ -176,11 +176,7 @@ void print_version()
 ProcessReturnCode parse_arguments(
```

Comment:
This function would make more sense as a method of `CommandlineArgs` extended in `CommandlineArgsRecorder`.